zabdig

```diff
@@ -459,6 +463,9 @@ class ZInterface(object):
 
         return js['result']
 
+    def _zabbix_version(self) -> str:
+        return self._zabbix_call("apiinfo.version", [], False)
```

Comment:
Can we turn this into a tuple of numbers here? Maybe at least the first digits?
```
version_str = self._zabbix_call("apiinfo.version", [], False)
return tuple((int(i) if i.isdigit() else i) for i in version_str.split('.'))
```
Or something.

See:
```
>>> version_str = '1.2.3a3.post3'
>>> tuple((int(i) if i.isdigit() else i) for i in version_str.split('.'))
(1, 2, '3a3', 'post3')
```
This makes it easier below to check versions.